### ISSUE TITLE
Don't include site name for specific sections

### ISSUE THREAD

**Issue by piotrpog** [vaersaagod/seomate#20]
With my homepage, I would like to have `title` without sitename appended.

I know i can disable specific settings in specific templates using Twig, like this:
```
{% set seomate = {
    config: {
        includeSitenameInTitle: false
    },
} %}
```
Can this be done using only config file?

**Comment by mmikkel** [comment]
No, the `includeSitenameInTitle` setting is not context aware, so the only way to do it is by using template overrides (as per your example).

I'll go ahead and label this issue as a feature request though.
